src/features/pages/home/live-pricing/table-component/live-market-table.tsx

```diff
@@ -10,58 +10,33 @@ import { TAvailableLiveMarkets, TMarketData } from '../types'
 import useLiveColumns from '../use-live-columns'
 import { table_row_header, table_row_data } from './live-pricing.module.scss'
 import Flex from 'features/components/atoms/flex-box'
-import useRegion from 'components/hooks/use-region'
 import Link from 'features/components/atoms/link'
 import { Localize } from 'components/localization'
+import useFetchFirebaseData from 'components/hooks/use-firebase-fetch-data'
 
 export type TLiveMarketTableProps = {
     selected_market: TAvailableLiveMarkets
     link_to: string
 }
 
 const LiveMarketTable = ({ selected_market, link_to }: TLiveMarketTableProps) => {
-    const { is_eu } = useRegion()
-    const [initial_loaded, setInitialLoaded] = useState(false)
+    const firebaseData = useFetchFirebaseData()
```

Comment:
If we make the change I suggested above, we can now return the 

```suggestion
    const [error, pricingData] = useFetchFirebaseData()
```

Then we can handle the `error` by maybe displaying an error, unless there is already a design for this.